recipes-bsp/acpi-tables/files/acpi-tables-load

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This script copies all ACPI tables into /kernel/firmware/acpi
+# You must have configfs enabled for this to work
+
+    for TABLE in `ls /kernel/firmware/acpi/ 2>/dev/null`; do
```

Comment:
Beeter to use `find /... -type f -name "*.aml"`.

Reply:
> Beeter to use `find /... -type f -name "*.aml"`.

Maybe leave it for now? Currently we can select which tables are built (ACPI_TABLES). But I can image where 2 tables are built and only one needs to be loaded by default. Like where LED's flash during configuration, and SPI is needed during normal operation. And we could unload SPI and load LED when entering configuration.

In that case we would need an additional configuration file, of pass list of tables to be loaded by doing `sed` on this script.

Reply:
I meant instead of `ls ...` better to use `$(find ...)`.

Reply:
Ah, then I will modify.